stdlib/InteractiveUtils/src/editless.jl

```diff
@@ -16,7 +16,11 @@ function editor()
     if Sys.iswindows() || Sys.isapple()
         default_editor = "open"
     elseif isfile("/etc/alternatives/editor")
-        default_editor = "/etc/alternatives/editor"
+        if islink("/etc/alternatives/editor")
+            default_editor = readlink("/etc/alternatives/editor")
+        else
+            default_editor = "/etc/alternatives/editor"
```

Comment:
duplicating the string here in the source seems unfortunate, but otherwise this fix sgtm
```julia
default_editor = "/etc/alternatives/editor"
islink(default_editor) && (default_editor = realpath(default_editor))
```

I've also changed `readlink` to `realpath` here, since the string returned by `readlink` may be a relative path, and it's simpler to use the `realpath` wrapper in libc than to reimplement the partial case here.

Reply:
How about just unconditionally doing `default_editor = realpath("/etc/alternatives/editor")`?